openqa-trigger-bisect-jobs

```diff
@@ -109,11 +109,14 @@ client_args = [
 
 
 def call(cmds, dry_run=False):
-    log.debug("call: %s" % cmds)
-    return subprocess.check_output(
-        (["echo", "Simulating: "] if dry_run else []) + cmds
-    ).decode("utf-8")
-
+    try:
+        log.debug("call: %s" % cmds)
+        return subprocess.check_output(
+            (["echo", "Simulating: "] if dry_run else []) + cmds
+        ).decode("utf-8")
+    except subprocess.CalledProcessError as e:
+        log.error("%s failed with return code %s", cmds[0], e.returncode)
```

Comment:
`e.returncode` might actually be a negative signal number. Then this error message might be confusing. Not a big deal but this could be taken into account.

There should probably be also a more generic `except` handler after this (maybe one that catches all kinds of exceptions) to handle other error cases as well, e.g. when the executable wasn't found or the execution wasn't permitted.

Reply:
I think all this goes way too far. For now we should focus on only the original problem about "Current job … will fail, because the repositories for the below updates are unavailable" because there we know what we should do